sdk/identity/azure-identity/CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Release History
 
+## 1.1.0-beta.4 (Unreleased)
```

Comment:
This happened since the current version listed is this without a changelog entry.